zerver/lib/outgoing_webhook.py

```diff
@@ -317,7 +317,17 @@ def do_rest_call(
             request_data=request_data,
         )
         if str(response.status_code).startswith("2"):
-            process_success_response(event, service_handler, response)
+            try:
+                process_success_response(event, service_handler, response)
+            except JsonableError as e:
+                response_message = (e.msg)
+                logging.exception("Outhook trigger failed:", stack_info=True)
```

Comment:
This is a user data error, not a server error; if we're going to log at all, it should only be at `info` at the highest.